demisto_sdk/commands/upload/tests/uploader_test.py

```diff
@@ -206,19 +206,60 @@ def test_upload_layout_positive(demisto_client_configure, mocker):
     Then
         - Ensure layout is uploaded successfully
         - Ensure success upload message is printed as expected
+        - Ensure that _unify isn't called.
     """
+    from demisto_sdk.commands.common.content.objects.pack_objects.layout.layout import (
+        LayoutObject,
+    )
+
     mocker.patch.object(demisto_client, "configure", return_value="object")
+    unify_mocker = mocker.patch.object(LayoutObject, "_unify")
+
     layout_name = "layout-details-test_bla-V2.json"
     layout_path = f"{git_path()}/demisto_sdk/tests/test_files/Packs/DummyPack/Layouts/{layout_name}"
+
     uploader = Uploader(input=layout_path, insecure=False, verbose=False)
     mocker.patch.object(uploader, "client")
     uploader.upload()
 
+    assert not unify_mocker.called
```

Comment:
just to make sure, is `.called` a property or a method?

Reply:
propery